### ISSUE TITLE
Move SOA values from Constants to profile

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-engine#945]
In `Constants.pm` the following SOA values are found:
```
Readonly our $SOA_DEFAULT_TTL_MAXIMUM_VALUE => 86_400;     # 1 day
Readonly our $SOA_DEFAULT_TTL_MINIMUM_VALUE => 300;        # 5 minutes
Readonly our $SOA_EXPIRE_MINIMUM_VALUE      => 604_800;    # 1 week
Readonly our $SOA_REFRESH_MINIMUM_VALUE     => 14_400;     # 4 hours
Readonly our $SOA_RETRY_MINIMUM_VALUE       => 3_600;      # 1 hour
```

Being there prevents the user from doing a simple policy change. It would be better to keep them in the policy.

The suggestion came from https://github.com/zonemaster/zonemaster/issues/972.

**Comment by Ressy66** [comment]
ok after playing merry-go-rounds with all these issue numbers constantly redirecting all over the place, i'll ask here one more time, is this implemented, if so what is the filename/location that is edited for custom values, it really shouldnt be this hard.
or do we just keep editing constants which seems simplest for those who have found it

**Comment by matsduf** [comment]
No, this is not yet implemented. There is not even a firm decision to have it implemented. I'm sorry, it will take some time.

As long as this issue is kept open it is not implemented. Any implementation will be done through a pull request that will be linked from this issue.

**Comment by matsduf** [comment]
A proposal is found in #1032.

**Comment by pnax** [comment]
Implemented with #1032
